src/Container/index.tsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable react/require-default-props */
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import RBContainer, { type ContainerProps as RBContainerProps } from 'react-bootstrap/Container';
+
+import type { ComponentWithAsProp } from '../utils/types/bootstrap';
+
+enum ContainerSizeClass {
+  xs = 'container-mw-xs',
+  sm = 'container-mw-sm',
+  md = 'container-mw-md',
+  lg = 'container-mw-lg',
+  xl = 'container-mw-xl',
+}
+
+export type ContainerSize = keyof typeof ContainerSizeClass;
+
+interface ContainerProps extends RBContainerProps {
+  size?: ContainerSize;
+}
+
+type ContainerType = ComponentWithAsProp<'div', ContainerProps> & { Deprecated?: any };
+
+const Container: ContainerType = React.forwardRef<HTMLDivElement, ContainerProps>(({
+  size,
+  children,
+  ...props
+}, ref) => (
+  <RBContainer
+    bsPrefix="container"
+    fluid
```

Comment:
nit: Do we need these lines when these properties are overridden by the prop defaults below?

I'm more concerned with the `fluid` one, since I'm unsure how overriding these boolean values works in Typescript, and don't want to break anyone else's UI.

```suggestion
```

Reply:
You're right. We don't need this.
Removed here: e96c4608b57a880ae31f4427c9c4e4791cc18ef0

As information only, because the spread `{...props}` was after these props, the spread would override it. The boolean `fluid` is a syntax sugar for `fluid={true}`. This is the transpiled JS code:

![image](https://github.com/user-attachments/assets/b3c842b1-c8ec-403a-ba05-3d07b678a417)